json-logs/samples/api/canvases.access.delete.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ok": false,
+  "error": "",
+  "response_metadata": {
+    "scopes": [
+      "canvases:write"
+    ],
+    "acceptedScopes": [ "canvases:write" ]
```

Comment:
These are manual additions by the node sdk: https://github.com/slackapi/node-slack-sdk/blob/%40slack/web-api%407.0.4/packages/web-api/src/WebClient.ts#L773-L779 

This means that **these properties actually do not exist in the response body**. They exists in request headers instead and the Java SDK provides a different way (`getHttpResponseHeaders()`) to access them.

Can you remove these before merging the changes?